packages/story-editor/src/app/api/apiProvider.js

```diff
@@ -41,10 +41,13 @@ function APIProvider({ children }) {
       currentUser,
       storyLocking,
       pageTemplates: customPageTemplates,
+      taxonomies,
+      taxonomyBase,
```

Comment:
`taxonomyBase` should not be retrieved from the config like this.

The full REST API URL for each taxonomy can be retrieved from the `_links` section of the `getTaxonomies()` response.

Reply:
awesome. Updated it to use that endpoint present in `_links`